ci/docker/runtime_functions.sh

```diff
@@ -399,6 +399,14 @@ unittest_ubuntu_gpu_R() {
     make rpkgtest R_LIBS=/tmp/r-site-library R_GPU_ENABLE=1
 }
 
+unittest_ubuntu_cpu_julia06() {
+    set -ex
+    julia -e 'versioninfo()'
+    julia -e 'Pkg.clone("julia", "MXNet")'
+    julia -e 'Pkg.build("MXNet")'
+    julia -e 'Pkg.test("MXNet")'
```

Comment:
Could you add a comment that points to the file which defines the test behaviour?

Reply:
Sure, I added some in 40ae566